app/presenters/hyrax/admin/users_presenter.rb

```diff
@@ -0,0 +1,33 @@
+module Hyrax
+  module Admin
+    class UsersPresenter
+      def initialize
+        @users = users
+      end
+
+      # @return [Array] an array of Users
+      def users
+        @users ||= search
+      end
+
+      # @return [Number] quantity of users excluding the system users and guest_users
+      def user_count
+        users.count
+      end
+
+      # @return [Array] an array of user roles
+      def user_roles(user)
+        roles = user.groups
+        return roles if roles.any?
```

Comment:
You're right, we no longer need to check it. At one point it was throwing errors in the view without the check, but I restructured that and apparently never went back to check if it was still needed. I've removed it.